### PR TITLE
New ipmi

### PR DESCRIPTION
**Pull request by coolo**
https://progress.opensuse.org/issues/19758

### CODE REVIEW

File: backend/ipmi.pm [hunk 1]
```diff
@@ -91,43 +91,21 @@ sub restart_host {
...
-
-    return 1;
-}
-
 sub do_start_vm {
     my ($self) = @_;
 
     # remove backend.crashed
     $self->unlink_crash_file;
+
```
Comment (sysrich) [inline backend/ipmi.pm]: NITPICK: Erroneous blank space?

File: backend/ipmi.pm [hunk 2]
```diff
@@ -91,43 +91,21 @@ sub restart_host {
...
     $self->restart_host;
-    $self->relogin_vnc;
 
     # truncate the serial file
     open(my $sf, '>', $self->{serialfile});
     close($sf);
 
-    my $sol = $testapi::distri->add_console('sol', 'ipmi-sol', {serialfile => $self->{serialfile}});
-    $sol->activate;
+    my $sol = $testapi::distri->add_console('sol', 'ipmitool-xterm');
```
Comment (sysrich) [inline backend/ipmi.pm]: Why 'ipmitool-xterm' instead of 'ipmi-xterm'? doesn't 'ipmi-sol' also use ipmitool? The weird naming confuses me.

Comment (coolo) [inline backend/ipmi.pm]: I used ipmitool initially as name - and then perlcritic complained that I need to leave out the '' in the classname hash. And I thought for 2 seconds about a better name containing a - for consistency :)

**Comment by coolo** [comment]
as discussed with Richard - merging for timely deployment

**Pull request merged**